Proposals/ParametricandOpticalAxes/ProposalSummary.md

```diff
@@ -0,0 +1,373 @@
+
+# Proposal: Parametric and Optical Axes
+
+## Administrative Information
+**Proposers name:** Sam Berlow
+
+**Vendor affiliation:** Type Network
+
+**Proposal name:** Parametric Axes
+
+**Date of submission:** 12/6/2017
+
+**New or revised proposal:** New
+
+**Previous revision date:** N/A
+
+## General Technical Information
+
+**Related axes:** See individual axis proposals
+
+**Similar axes:** See individual axis proposals
+
+**Axis type:** Parametric
+
+## Proposed Axis: `xtra`
+
+**Tag:** xtra
+
+**Name:** x transparent
+
+**Axis type:** Optical
+
+**Description:** assigns a “white” per mille value to each instance of the design space
+
+**Valid numeric range:**  -1000 to 2000
+
+**Scale interpretation:** Values should be interpreted as per-mille-of-em
+
+**Recommended or required “Regular” value:** N/A
+
+**Suggested programmatic interactions:** Example: by setting a maximum word space parameter, and adjusting the XTRA units to a min and max. A developer could adjust justified text without tracking letters. Maintaining the apparent weight and width. 
+
+**UI recommendations:** Users may choose to program a variant in connection to direct or conjunctive input for a page description language, or via a user interface
+
+**Script or language considerations:** Many languages and scripts 
+
+**Related axes:** wght, width, opsz
+
+**Similar axes:** spac
+
+**Additional information:** XTRA changes the white space in the x or horizontal direction. This useful in adjusting type horizontally without changing the opaque (black) Useful in VR, TV, and justification of text blocks.
+
+**Conventionality benefits:** "White space" or "Negative Shapes" are common concepts in visual arts, but many users don't think about the transparent area of type deconstructed (or essentially isolated) to the X dimension. This axes helps users to structure their thinking about white space horizontally, for the overall/total typeface.
+
+**Interoperability benefits:** justification, legibility
```

Comment:
Interop has to do with the semantics of an axis being commonly understood and working the same across different fonts. So, for this field, what's expected in some explanation of how having conventional semantics will benefit uses, or font developers. For example, in this case, I think the suggestion is that xtra could be used for programmatic justification since the effect on metrics by changing this axis will be, to some degree, predictable. If so, then that's a benefit you should describe, rather than simply "justification". (If you just say "justification", the question will probably just come back later as to how this will facilitate justification.)